events/admin.py

```diff
@@ -1,7 +1,62 @@
+from django import forms
 from django.contrib import admin
 
 from .models import Event, Tag, VideoAsset
 
-admin.site.register(Event)
-admin.site.register(VideoAsset)
+
+class VideoAssetForm(forms.ModelForm):
```

Comment:
It might be a good idea to move this form class to forms.py for better organization.